### DISCUSSION TITLE
Problème avec ddt4all

### DISCUSSION THREAD

**Discussion by just4545** [cedricp/ddt4all#876] [Q&A]
Bonjour

Je me permet de créer ce ticket afin de vous informer que je rencontre un problème ,j'ai installé votre dernière version de ddt4all -> https://github.com/cedricp/ddt4all/releases/tag/v2.1.1 , une fois installé je lui ai importé la base de donnée jusque la tout est OK.

J'utilise et obdII en WIFI ->https://www.amazon.fr/dp/B01NAQ2878?psc=1&ref=ppx_yo2ov_dt_b_product_details, la connexion avec ddt4all se fait correctement. Si j'utilise ddt4all c'est seulement pour activer l'Android Auto sur un R-link.

J'ai bien verifié que mon R-Link était bien a la deniere version etc...

Parcontre quand je fais des modifs et que je les applique rien ne se passe ou meme juste un simple reset de mon poste radio sa ne fonctionne pas et  sa reste bloquer sur le screen d'en dessous : 

![image](https://github.com/cedricp/ddt4all/assets/140565077/8b79a772-2ed5-45fd-878c-2b5779efb443)

et ça fait rien après, pourriez m'aider s'il vous plait 

Merci d'avance

**Comment by AndyIreland** [comment]
try pressing the 'home' button to the right of the touchscreen for the rlink   5 times and see if it reboots the rlink and then see if AA has installed

**Comment by Furtif** [comment]
Pour activer AA je crois que aller dans micom suffit si c'est le même principe que media nav.
Par autre coté je crois que les fonctions multimédia sont accessibles via CAN2 et un OBD2 basic n'a pas accès a ce coté la mis a part en croisé, voir https://github.com/cedricp/ddt4all/wiki#how-to-access-multimedia-can-bus

**Reply by lionelsoron7-create** [reply]
Bonjour, avez vu réussi. Je voudrais coder mon écran multimédia mais ça marche pas. Et mon Dtt4all n’a pas le mode expert. Je comprends pas.

**Comment by just4545** [comment]
> Pour activer AA je crois que aller dans micom suffit si c'est le même principe que media nav. Par autre coté je crois que les fonctions multimédia sont accessibles via CAN2 et un OBD2 basic n'a pas accès a ce coté la mis a part en croisé, voir https://github.com/cedricp/ddt4all/wiki#how-to-access-multimedia-can-bus

**Comment by just4545** [comment]
Bonjour @Furtif 

Merci d'avoir pris le temps de me répondre , pourrais-tu me dire c'est quoi micom ? car j'ai suivi cette video : https://www.youtube.com/watch?v=ZXObC7j4WtM&pp=ygUeYWN0aXZhdGUgYW5kcm9pZCBhdXRvIGluIHJsaW5r et il utilise bien le obd2 que j'ai acheté mais je comprends pas pourtant le obd2 se connecte bien a mon pc etc.. mais il arrive pas a écrire ce que je lui demande.

**Comment by Furtif** [comment]
Avais vous cliquer sur l’icône d’Albert pour passer en mode expert ? 

Pour micom voir https://m.youtube.com/watch?v=5pElphWMa9o

https://www.motorcarsoft.com/viewtopic.php?t=17928

**Comment by just4545** [comment]
Bonjour @Furtif 

Oui j'ai bien cliqué sur le bouton d'Albert mais sur le coté droit il y a marqué NO DATA sur tous et quand je fais read all sa ne fait rien

J'ai changé de prise j'utilise cette prise maintenant : https://diagauto-store.fr/products/valise-de-diagnostic-multimarque-ds-150e-en-francais-bt-usb?gclid=EAIaIQobChMIlfLZ252vgAMVSEBBAh1pKwnsEAQYAiABEgKA7vD_BwE   sa devrait quand meme fonctionner avec ddt4all ?

**Comment by Furtif** [comment]
> Bonjour @Furtif
> 
> Oui j'ai bien cliqué sur le bouton d'Albert mais sur le coté droit il y a marqué NO DATA sur tous et quand je fais read all sa ne fait rien
> 
> J'ai changé de prise j'utilise cette prise maintenant : https://diagauto-store.fr/products/valise-de-diagnostic-multimarque-ds-150e-en-francais-bt-usb?gclid=EAIaIQobChMIlfLZ252vgAMVSEBBAh1pKwnsEAQYAiABEgKA7vD_BwE sa devrait quand meme fonctionner avec ddt4all ?

je ne sais pas... mais c'est pas dit normalement si a tous a NO_DATA c'est car vous n'avais pas accès au coté CAN2 comme dit ici https://github.com/cedricp/ddt4all/wiki#how-to-access-multimedia-can-bus pour y avoir acces a ce coté vous faut une sonde Derelek ou Bosh car on 2 modes http://www.derelek.com/nos-produits/ (normalement USB-DIAG 3) ou croiser les pins comme dit dans le wiki du github voir le site russe que explique comment faire.

Votre DDT4All est bon faut just que vous trouvais une solution pour rentrer dans coté multimedia CAN2

**Comment by Furtif** [comment]
![image](https://github.com/cedricp/ddt4all/assets/11375035/dd631f02-f133-443a-803f-9d770e8dfc0f)

comme vous pouvais le voir il y a 2 modes (en gros il croise le pin) mode Diag (multimédia CAN2) e mode véhicule.

**Comment by just4545** [comment]
Et vous pensez que avec celui-là va fonctionner : https://www.amazon.fr/dp/B07MQ8GHG3?psc=1&ref=ppx_yo2ov_dt_b_product_details

**Comment by Furtif** [comment]
> Et vous pensez que avec celui-là va fonctionner : https://www.amazon.fr/dp/B07MQ8GHG3?psc=1&ref=ppx_yo2ov_dt_b_product_details

Honnêtement je peut pas vous le dire car je ne l'ai pas tester. Désolé.

**Comment by just4545** [comment]
ça marche, je vais tester et puis je vous dirai
Merci pour votre aide :)

**Comment by Furtif** [comment]
de tout façon vous aller avoir accès que coté véhicule même que ce câble marche, comme je vous ai expliquer pour accès CAN2 y a que 2 solutions soit utiliser la sonde derelec destiné d'origine a DDT2000 (a savoir que ddt4all utilise les database de ddt2000 et est basé sur ce logiciel) soit utiliser un OBD2 comme lui 

![image](https://github.com/cedricp/ddt4all/assets/11375035/60827058-3e75-4fda-8b98-8cf09909ab31)

et lui faire une manipulation pour avoir les 2 modes

![image](https://github.com/cedricp/ddt4all/assets/11375035/4bd866e7-7414-458e-a485-39edbc206b94)


voir https://www-drive2-ru.translate.goog/l/468057702971474426/?_x_tr_sl=auto&_x_tr_tl=fr&_x_tr_hl=fr&_x_tr_pto=wapp (je mise le site dans google translate pour lire un peut mieux).

**Comment by just4545** [comment]
oui alors celui que je vous ai montré : https://www.amazon.fr/dp/B07MQ8GHG3/ref=pe_27091421_487030221_TE_SCE_dp_1 il a l'interrupteur donc je pense qu'il va pouvoir activer le mode en question

**Comment by Furtif** [comment]
ou utiliser une rallonge découper les fils et un peut de scotch lol.

voir cette vidéo sa pourra vous éclairer. https://www.youtube.com/watch?v=iDfbmN-OfdM

**Comment by Furtif** [comment]
> oui alors celui que je vous ai montré : https://www.amazon.fr/dp/B07MQ8GHG3/ref=pe_27091421_487030221_TE_SCE_dp_1 il a l'interrupteur donc je pense qu'il va pouvoir activer le mode en question

oui a l'interrupteur, voir... comme j'ai pas tester je sais pas. testez puis venais nous le dire.

**Comment by just4545** [comment]
ça marche pas de soucis , je vous tiens au courant 
Merci pour tout :)

**Comment by Furtif** [comment]
vais changer l'emplacement de cette conversation vais déplacer dans discutions car ceci n'est pas un issue logiciel mais un problème matériel.

**Reply by just4545** [reply]
Alors je viens de faire le test ça ne.fonctionne pas donc je vais devoir acheter la rallonge obd2 mais ma question est est-cequil y a une rallonge que je peux acheter ou il y a pas besoin de chabger l'emplacement des pins ?
